### PR TITLE
Basic admin area

### PR DESCRIPTION
**Pull request by jongilbraith**
Bootstrap based layout. Fairly comprehensive navigation and searching of firms, advisers, principals and lookup data.

### CODE REVIEW

File: app/controllers/admin/lookup_advisers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Admin::LookupAdvisersController < Admin::ApplicationController
```
Comment (benlovell) [inline app/controllers/admin/lookup_advisers_controller.rb]: Can we namespace these `LookupXXXController` under `Lookup` and take it out of the controller names please?

Comment (jongilbraith) [inline app/controllers/admin/lookup_advisers_controller.rb]: Yep, no worries. Had considered that myself.

File: config/routes.rb [hunk 2]
```diff
@@ -26,4 +26,15 @@
...
 
   resource :contact, only: :create
+
+  namespace :admin do
+    root 'pages#home'
+    resources :advisers, only: [:index, :show]
+    resources :firms, only: [:index, :show] do
+      resources :advisers, only: :index
+    end
+    resources :lookup_advisers, only: :index, path: 'lookup-advisers'
```
Comment (benlovell) [inline config/routes.rb]: Once these are namespace we won't need to explicitly set the `path` either. I know you're aware just putting it down for completion sake :smile:

**Comment by jongilbraith** [comment]
Lookup namespace sorted, will do the basic auth first thing Monday.

File: app/controllers/admin/lookup/advisers_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Admin::Lookup::AdvisersController < Admin::ApplicationController
+  def index
+    @search = ::Lookup::Adviser.ransack(params[:q])
```
Comment (jongilbraith) [inline app/controllers/admin/lookup/advisers_controller.rb]: Would be interested to get to the bottom of this, but for some reason ``Lookup::Firm`` resolves fine within ``Admin::Lookup::FirmsController``, but ``Lookup::Adviser`` doesn't within ``Admin::Lookup::AdvisersController`` and I have to be explicit about the namespace.

**Comment by benlovell** [comment]
:+1:

**Comment by tomas-stefano** [comment]
:+1: :+1:

**Pull request merged**